app/models/manageiq/providers/ovirt/infra_manager/provision/cloning.rb

```diff
@@ -10,11 +10,8 @@ def destination_image_locked?
   end
 
   def find_destination_in_vmdb(ems_ref)
-    if source.template?
-      ManageIQ::Providers::Ovirt::InfraManager::Vm.find_by(:name => dest_name, :ems_ref => ems_ref)
-    else
-      ManageIQ::Providers::Ovirt::InfraManager::Template.find_by(:name => dest_name, :ems_ref => ems_ref)
-    end
```

Comment:
The old way did not include the ems_id in the query and could identify a VM on another EMS :cry: